pkg/routing/redisrouter.go

```diff
@@ -237,82 +173,9 @@ func (r *RedisRouter) WriteNodeRTC(_ context.Context, rtcNodeID string, msg *liv
 	return r.writeRTCMessage(rtcSink, msg)
 }
 
-func (r *RedisRouter) startParticipantRTC(ss *livekit.StartSession, participantKey livekit.ParticipantKey, participantKeyB62 livekit.ParticipantKey) error {
-	prometheus.IncrementParticipantRtcInit(1)
```

Comment:
These are used to track join success rate IIRC. Is there a new place where this happens? Or we need to clean those stats up. I think those stats have not been tricky to track.

Reply:
this was the legacy path. this happens in pkg/service/signal.go now